From e2d32e3385e2d51bc6401a1eecc725edfe785665 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 2 Sep 2016 13:05:29 +0200
Subject: [PATCH 0507/1345] arm64: dts: marvell: add pinctrl description for
 rgmii

commit  952c531ba6ec4ce9430535f657e14cd550fdd5bb from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Placing pinctrl description, inside one of the child node of 'ppv22' would
be more clear. Unfortunately pinctrl description can not be moved to the
child nodes of 'ppv22', because the pinctrl framework requires that the
description is placed inside the device nodes (it is because all pinctrl
routines are called during driver_probe_device). In case of ppv2, there is
only one driver registered for whole ppv22, which is common for each port.

Change-Id: Ia94bd7cd7438cfe3df2029d5bd17cb6af7ea71cc
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32934
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-8040-db-router.dts     |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
index 1995ad2..2216d3f 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
@@ -90,6 +90,9 @@
 				};
 			};
 			ppv22@000000 {
+				pinctrl-names = "default";
+				pinctrl-0 = <&cpm_ge2_rgmii_pins>;
+
 				eth0: eth0@010000 {
 					status = "okay";
 				};
@@ -130,6 +133,9 @@
 				};
 			};
 			ppv22@000000 {
+				pinctrl-names = "default";
+				pinctrl-0 = <&cps_ge1_rgmii_pins>;
+
 				eth0_1: eth0@010000 {
 					status = "okay";
 				};
-- 
1.7.9.5

